index.html

```diff
@@ -5,9 +5,40 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <link rel="stylesheet" href="style.css">
-    <title>Document</title>
+    <title>Portfolio</title>
 </head>
 <body>
-    <h1>hello</h1>
+    <div id="header-background-mobile">
+      <header id="Toolbar-Dark-Compact">
+         <div id="Logo-Placeholder">
+             <p id="My-Logo">Qwerty</p>
+         </div>
+         <div id="Normal-Button-Tertiary-Enabled">         
+            <div id="icon-menu"><img src="img/NormalButton/Tertiary/Icons/Union.png" alt="Menu"></div>
+         </div>
+      </header>
+      <div id="Headline">
+         <h2 id="Primary-Text">I’m Danys
+             Glad to see you!
+         </h2>
```

Comment:
- [ ] Kindly change your header tag to h1 since this is your first header tag, this will make your page optimized for SEO.